pages/common/git-commit.md

```diff
@@ -11,7 +11,7 @@
 
 `git commit -a -m {{message}}`
 
-- Replace the last commit with currently staged changes:
+- Add currently staged changes on the last commit:
```

Comment:
Hrm, I see your point. However, the new description here implies that commits can be changed without altering their hash value. What about this?
```suggestion
- Update the last commit by adding the currently staged changes (this changes the commit's hash):
```